### ISSUE TITLE
How to set training as False in dropout during eval

### ISSUE THREAD

**Issue by qibin-google** [chiphuyen/tf-stanford-tutorials#125]
https://github.com/chiphuyen/stanford-tensorflow-tutorials/blob/51e53daaa2a32cfe7a1966f060b28dbbd081791c/examples/07_convnet_layers.py#L146

I don't understand how setting self.training = False turns training off in dropout. self.training is passed as an argument when building dropout layer. After the whole network is built, changing self.training should not change the network.
